crates/cli/tests/app_e2e.rs

```
@@ -24,7 +24,8 @@ fn test_cli_app_e2e() -> Result<()> {
             "--exe-output",
             temp_exe.to_str().unwrap(),
         ],
-    )?;
+    )
+    .unwrap_or_else(|e| println!("{:?}", e));
```

is this intentional? it means it won't error on failure

---

Ah yes, this should `panic!` instead of `println!`